src/services/errors.js

```diff
@@ -12,5 +12,14 @@ function InvalidFiltersFormatError(message) {
 }
 InvalidFiltersFormatError.prototype = new Error();
 
+function InvalidParameterError(message) {
+  this.name = 'InvalidParameterError';
+  this.message = message || 'The given parameter is invalid.';
+  this.status = 422;
+  this.stack = (new Error()).stack;
+}
+InvalidParameterError.prototype = new Error();
```

Comment:
This is a bad evolution of the code in my opinion, started by `NoMatchingOperatorError` creation.
1. Code duplication (I hope we can do something against it)
2. Those 3 errors will generate HTTP 422 errors (with a different payload message), I really prefer `ErrorHTTP422` (vs `InvalidParameterError`) because as soon as you read the error, you'll know what will be the service response: 422.

Reply:
I'm not sure what to do with this one. I added one new type of error over the existing ones (there is no 422 base in this repository). Basically, each error is written like the one below (I highlighted the duplication part)

```javascript
function InvalidParameterError(message) {
  this.name = 'A NAME';
  this.message = message || 'A MESSAGE';
  this.stack = (new Error()).stack; // <-- Duplication
}
InvalidParameterError.prototype = new Error(); // <-- Duplication
```

I did not created error like this until now, but when I read the existing code (that I decided to extend) I thought it was OK to add one following the same pattern. 2 duplicated lines could be considered preferable over abstraction.

@arnaudbesnier I don't know how to fix this one. Could you help me?

Reply:
Aslo see: https://github.com/ForestAdmin/forest-express-sequelize/pull/376#discussion_r376308390

Reply:
Sorry my bad, I thought it was the same code as `forest-express-sequelize`.
Let's forget it